doc/manual/manual.tex

```diff
@@ -1508,9 +1508,17 @@ \subsubsection{Almost linear models}
 instead in each time step solves first the Stokes system with the previous
 time step's temperature, and then uses the so-computed velocity to solve the
 temperature equation. These kind of time stepping schemes are often referred
-to as \textit{IMPES} methods (they originate in the porous media flow
+to as \textit{operator splitting} methods. A particular operator
+splitting method, used in
+earlier \aspect{} versions, solves first the Stokes equations and then
+uses an explicit time stepping method for the temperature equation;
```

Comment:
Only the convective term is explicit, so this is not "an explicit time stepping method". Can you please fix this?